packages/frontend-shared/src/gql-components/OpenConfigFileInIDE.vue

```diff
@@ -7,14 +7,15 @@
     >
       <slot>
         <span
-          class="cursor-pointer text-purple-500"
+          class="text-purple-500 cursor-pointer"
         >
           {{ configFile }}
         </span>
       </slot>
     </button>
 
     <ChooseExternalEditorModal
+      v-if="isChooseEditorOpen"
```

Comment:
I am not sure this is part of the create spec modal but eh... I am glad someone did it \o/

Reply:
The create-spec-flow uses the specPattern component -> OpenConfigFileInIDE